specs/jsonschema-core.md

```diff
@@ -415,6 +415,12 @@ keywords MUST NOT begin with this prefix.
 Implementations MUST refuse to evaluate schemas which contain keywords which
 they do not know how to process or explicitly choose not to process.
 
+## Specification Versioning and Compatibility
+
+This specification is versioned by two values: iteration and release year.
```

Comment:
The term "iteration" doesn't feel right to me. It's not very intuitive and it's never defined what it means.

Reply:
I had "iteration" at first because I wanted to avoid "version" since it seems it be such an overused term.  But rereading it a few days ago, it didn't feel right.  I thought I got all of them.